pkg/util/child.go

```diff
@@ -0,0 +1,30 @@
+// +build !linux
+
+/*
+Derived from:
+
+https://github.com/kubernetes/kubernetes
+
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"os"
+)
+
+// childSignals are the allowed signals that can be sent to children in Windows to terminate
+var childSignals = []os.Signal{os.Interrupt, os.Kill}
```

Comment:
@bparees I don't understand this decision, why catch SIGKILL in `!linux` platforms, and not catch it on `linux`?!

According to [`os/signal` docs](https://golang.org/pkg/os/signal/#hdr-Types_of_signals):

> The signals SIGKILL and SIGSTOP may not be caught by a program, and therefore can not be affected by this package.

I believe the existence of `os.Kill` is to send it to processes, not to catch it. See https://golang.org/pkg/os/#Process.Signal